agent/bench-scripts/pbench-fio

```diff
@@ -139,12 +140,14 @@ function fio_usage() {
 		printf "\t\tDefault is n\n"
 		printf -- "\t--remote-only\n"
 		printf "\trun this on the remotes only\n"
+		printf "\t--histogram-interval-msec=<int>\n"
+		printf "\tset the histogram logging interval in milliseconds (default 10000)\n"
```

Comment:
Sporry, missed this: this should be 

    printf "\tset the histogram logging interval in milliseconds (default $histogram_interval_msec)\n"